Added file for turtles implementation game

jazbassi1 opened the pull request.

jazbassi1 committed:
Add files via upload

manokhina reviewed (commented):
It's hard to understand what the PR is about. Let's aim to add more documentation and comments into the code and the PR itself. Imagine I have completely no idea what the PR is about (in fact I really don't!). What would you like to see in it to understand what the code is doing if you were me?

jazbassi1 committed:
Update turtle-square.py

added implementation of the turtle module of a turtle moving and drawing a square
